### DISCUSSION TITLE
The possibility to add more than one shader at the same time.

### DISCUSSION THREAD

**Discussion by armentix** [mausimus/ShaderGlass#84] [Ideas]
As the title says. It would be very good if we could do that !

Also i noticed HDR content don't pass through the glass. Colors are washed out.

BTW very good software ! I was serching for something like that for years !

**Comment by gakun17** [comment]
Same here, I managed to combine a few of the filters on Retroarch to fit what I wanted, but that doesn't seem possible on ShaderGlass.

VLC has a rather pathetic VHS filter, I couldn't figure out how to install MPV, PlexHTPC is very not what I wanted and the FFMpeg in Retroarch barely works, often crashing when switching audio tracks and it's just very clunky overall with pausing and seeking.

ShaderGlass is such a simple solution, I hope we'll be able to combine shaders in the future!

**Comment by OkayHero** [comment]
If you save a preset within RetroArch as a .slangp file, you can import it into ShaderGlass, this is the only workaround I've found, but it would be nice to be able to do it in app.

**Comment by isamu99** [comment]
Great and fantastic suggestion, gentlemen. This, along with an autolaunch feature with your preferred game/application option would make me stop using reshade altogether.

**Comment by MrBlastman** [comment]
OP:  You can already do this through the MegaBezel implementation.  I have custom 42 - 47 pass shaders, this way.  MegaBezel is incredibly powerful, and can produce results such as this:

https://i.imgur.com/SBfJMUq.png

https://i.imgur.com/qFLy19M.png

https://i.imgur.com/tcGFYkW.jpeg

You simply develop the shader presets in Retroarch, save them, and then import them directly into ShaderGlass.

Only issue is ShaderGlass HDR implementation is very weak, presently, so they can only be used in SDR, for now.

**Comment by BouncyBonBon** [comment]
would be incredible for the shaders like the denoiser and edge detection that don't do much on their own but are the key components of like every shader ever made
